docs/_config.yml

```diff
@@ -64,8 +67,10 @@ sphinx:
         - jupytext.reads
         - fmt: Rmd
     bibtex_reference_style: author_year  # or label, super, \supercite
-    execution_show_tb: True
-    execution_timeout: 120
+    # unknown_mime_type - application/vnd.plotly.v1+json and application/vnd.bokehjs_load.v0+json
+    # domains - sphinx_proof.domain::prf needs to have `resolve_any_xref` method
+    # mime_priority - latex priority not set in myst_nb for text/html, application/javascript
+    suppress_warnings: ["mystnb.unknown_mime_type", "myst.domains", "mystnb.mime_priority"]
```

Comment:
@AakashGfude are these warnings because they aren't supported `mime` types in the docs? This release should be able to support everything the current `jupyter-book` release does right?

Reply:
These mime types mentioned are not specified in the default latex mime list in myst-nb : https://github.com/executablebooks/MyST-NB/blob/master/myst_nb/core/render.py#L1195. We can add them in the _config here. Or in a separate PR along with testing out the output.

Reply:
@chrisjsewell any suggestions with these warnings. I'm not a huge fan of suppressing families of warnings, so I wonder if we should specify the `mime` type order for the `docs`.